pom.xml

```diff
@@ -316,6 +317,7 @@
                     <version>${bnd-maven-plugin.version}</version>
                     <configuration>
                         <bnd><![CDATA[Export-Package: oshi.*;-noimport:=true;-split-package:=merge-first
+                        Import-Package: ${osgi.slf4j.import.packages}, *
```

Comment:
I had assumed it was similar to this!  Does the `-noimport:=true` line above have any relevance here?

Reply:
This directive is for "Export-Package" only as I understand it, from the javadocs: `The package must not be substitutably imported.`
Quick search led me to this article https://bnd.bndtools.org/heads/export_package.html. So I'm not sure this is needed in "Import-Package".